src/Mcfedr/AwsPushBundle/Command/EnableAllCommand.php

```diff
@@ -6,21 +6,21 @@
 
 use Aws\Sns\SnsClient;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+#[AsCommand('kidslox:account:mcfedr:aws:enable', 'Reenable all devices')]
```

Comment:
The command name has changed with this commit. Is it expected?

Reply:
@slc1 its weird the command names where changed. I think that needs reverting

Reply:
I can make a PR to fix that

Reply:
Here we go: #48